test/parallel/test-fs-stream-fs-options.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+const path = require('path');
+const fs = require('fs');
+const assert = require('assert');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+const writeStreamOptions = ['open', 'close', 'write'];
+const readStreamOptions = ['open', 'close', 'read'];
+
+{
+  const file = path.join(tmpdir.path, 'write-end-test0.txt');
+
+  writeStreamOptions.forEach((fn) => {
+    const original = {
+      fs
+    };
```

Comment:
Nit: `original` is being defined three times without being modified, could you please define this one on the main scope and name it as `originalFs` or something like that?